### ISSUE TITLE
String is outside of file boundary error

### ISSUE THREAD

**Issue by boshkash1151** [questdb/questdb#3809]
### Describe the bug

When trying to execute a simple select query, I get the following error:

`error [msg=`String is outside of file boundary [offset=48, len=3145777, size=313114]`, errno=0, q=`select ip from`

### To reproduce

1. Go to http://serverip:9000
2. Execute any select query 
3. Get this error : `error [msg=`String is outside of file boundary [offset=48, len=3145777, size=313114]`, errno=0,`

### Expected Behavior

query to return 10 records

### Environment

```markdown
- **QuestDB version**: 7.3.2 
- **OS**: docker image : questdb/questdb:7.3.2 run on arm64
- **Browser**: Chrome Version 117.0.5938.132
```

**Comment by puzpuzpuz** [comment]
This looks like a corrupted database file. Could it be the case that your server experienced power loss? If that's the case and that's possible in future, you should configure sync commit mode:
https://questdb.io/docs/concept/storage-model/#consistency-and-durability

**Comment by boshkash1151** [comment]
@puzpuzpuz yeah it has experienced power loss few days ago. I will enable sync mode, but is there a way to recover old data ?

**Comment by ideoma** [comment]
Here is a couple of workarounds:

- You can narrow down the corruption to a partition and then drop the partition. 
- Update the column at the corrupted partition to `null`

**Comment by boshkash1151** [comment]
@ideoma unfortunately upon executing show PARTITIONS from 'table_name', I found that there is only one partition that holds everything.
